lib/index.js

```diff
@@ -469,10 +479,8 @@ api.updateRoles = brCallbackify(async (
   // 2. add specified resource roles
   // 3. ensure resource roles are unique
   const resourceRoles = _.uniqWith(
-    brPermission.mergeCapabilities(
-      brPermission.subtractCapabilities(meta.sysResourceRole, remove),
-      add),
-    _.isEqual);
+    brPermission.mergeCapabilities(brPermission.subtractCapabilities(
+      meta.sysResourceRole, remove), add), _.isEqual);
```

Comment:
Need this last one changed back -- it was much more readable before.

Reply:
going to look in master and restore it to that.